config/webpack.common.js

```diff
@@ -287,7 +286,6 @@ module.exports = function (webpackEnv, args) {
                 tsBuildInfoFile: paths.misc.tsBuildInfoFile,
               },
             },
-            context: paths.sources.srcFolder,
```

Comment:
Definiert, dass normalerweise TypeScript ausgehend von `./DistributionPackages` über TypeScript Files linted. Das ist in einem normalen Projekt ein nettes Extra, aber macht effektiv keinen Unterschied. Hingegen für Projekte, die aus dem Aufbau rausfallen (bspw. Mono-Repo) sorgt dies dafür, dass viel zu viele Dateien gelinted werden. Kann man auch drin lassen und in mono-Repos auf `undefined` setzen - ich finds so rum besser, aber lass drüber sprechen.